proto/requests.proto

```diff
@@ -131,3 +131,8 @@ message GetAccountStateDeltaRequest {
     // Block number up to which the delta is requested (inclusive).
     fixed32 to_block_num = 3;
 }
+
+message GetAccountStatesRequest {
+    // List of account IDs to get states.
+    repeated account.AccountId account_ids = 1;
+}
```

Comment:
Let's add something like `iinclude_headers` flag (false by default). When set to false, `AccountStateResponse.state_header` would be omitted.